transport-blockhound-tests/src/test/java/io/netty/util/internal/NettyBlockHoundIntegrationTest.java

```diff
@@ -82,8 +83,13 @@
 import static org.junit.jupiter.api.Assertions.fail;
 import static org.junit.jupiter.api.Assumptions.assumeTrue;
 
+@DisabledIf("isDisabledIfJavaVersion18OrAbove")
 public class NettyBlockHoundIntegrationTest {
 
+    private static boolean isDisabledIfJavaVersion18OrAbove() {
+        return PlatformDependent.javaVersion() >= 18;
```

Comment:
Oh, why does this no longer work?

Reply:
Their version of byte-buddy does not support java18 and older. Alternative we could also exclude theirs and add a working one. I thought for now we can just disable the test.